client/components/LayoutEditor/layoutEditor.scss

```diff
@@ -46,6 +46,19 @@
 			max-width: 100%;
 			padding-top: 12px;
 		}
+		.controls-row {
+			display: flex;
+			margin: 0 0.5em;
+			& > .input-field-component {
+				margin: 0;
+			}
+			& > *:not(:last-child) {
+				margin-right: 0.5em;
+			}
```

Comment:
I think the `&` might be redundant here. You should be able to remove them for these two rulesets, but not the `&:not` below.